plugins/fields/media/media.php

```diff
@@ -39,7 +39,12 @@ public function onCustomFieldsPrepareDom($field, DOMElement $parent, Form $form)
 		}
 
 		$fieldNode->setAttribute('type', 'accessiblemedia');
-
+		
+		if (JFactory::getApplication()->getIdentity()->authorise('core.create', 'com_media'))
```

Comment:
```suggestion
		if (Factory::getApplication()->getIdentity()->authorise('core.create', 'com_media'))
```

and add at the top of the file
`use Joomla\CMS\Factory;`